packages/api/test/e2e/doughnut-v0.e2e.ts

```diff
@@ -64,9 +65,11 @@ describe('Doughnut for CennznetExtrinsic', () => {
   let keyring;
 
   beforeAll(async () => {
-    // TODO: Update to Rimu
-    api = await Api.create({ provider: 'ws://localhost:9944' });
-    keyring = testingPairs({ type: 'sr25519' });
+    await cryptoWaitReady().then(async () => {
```

Comment:
if you are using await, no need to have `.then`

Reply:
had to be extra sure